src/f4enix/output/mctal.py

```
@@ -1378,3 +1382,108 @@ def _get_dfs(
+    user_label : str, optional
+        The name of the user column to normalize over (default is "User").
+    inplace : bool, optional
+        If True, modifies the DataFrame in place. If False, returns a new DataFrame.
+
+    Returns
+    -------
+    pd.DataFrame
+        DataFrame with an added 'Normalized Value' column.
+    """
+    if user_label not in df.columns:
+        raise ValueError(f"No '{user_label}' column found in DataFrame.")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)